.github/workflows/test-spec.yml

```
@@ -36,6 +36,10 @@ jobs:
             tests: routing
           - version: 2.18.0
             tests: snapshot
+          - version: 2.16.0
```

```suggestion
          - version: 2.18.0
```

Let's just use the latest, update 2.16 here to 2.18 and remove 2.17.